### PR TITLE
docs(select-menu, skeleton, stack): DLT-1798 move sb to docs 

### PR DESCRIPTION
**Pull request by ninamarina**
# docs: DLT-1798 move sb to docs - Scroller to Table

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket
[DLT-1798]

## :book: Description
Moves SB to docs for the following components:

- Select menu
- Skeleton
- Stack

[DLT-1798]: https://dialpad.atlassian.net/browse/DLT-1798?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/dialtone-documentation/docs/components/select-menu.md [hunk 1]
```diff
@@ -350,77 +263,184 @@ vueCode='
...
+</dt-select-menu>
+'
+/>
+
 ## Sizes
 
 We offer different sizes for instances in which the interface requires a smaller or larger select. In general, though, use the base (medium) size select as much as possible, especially in forms.
 
-<code-well-header bgclass="d-bgc-primary">
+<code-well-header bgclass="d-bgc-primary" class="d-d-block">
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/select-menu.md:437]: Seems like the "medium" case is not showing up for the vue and html example

Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/select-menu.md:437]: Added those examples

**Comment by ninamarina** [comment]
@francisrupert @juliodialpad please review whenever you can

**Comment by juliodialpad** [comment]
The title says `Scroller to Table` but you're only moving "Select menu, Skeleton and Stack" why are the Scroller and the Table components left out?

**Review by juliodialpad — APPROVED** [review]
Code-wise looks good, please update the PR title to match what you migrated here.

Defer to @francisrupert or @fede-dp to review content-wise if necessary but as you moved the docs over, I wouldn't worry too much about it.

Thanks!

**Comment by ninamarina** [comment]
> The title says `Scroller to Table` but you're only moving "Select menu, Skeleton and Stack" why are the Scroller and the Table components left out?

Oh, it's because the Scroller and the Table were already on the docs and not on SB, so there was nothing to migrate. Will update the title 👍

**Pull request merged**